core/src/map.rs

```diff
@@ -17,9 +18,10 @@ pub struct MapHandle(pub Handle<MapMeta>);
 #[ulid = "01GP3Z38HKE37JB6GRHHPPTY38"]
 pub struct MapSpawned(pub bool);
 
-#[derive(Clone, TypeUlid, Default, Deref, DerefMut)]
+/// Component defining the entity's spawner entity
+#[derive(Clone, TypeUlid, Deref, DerefMut)]
 #[ulid = "01GP9NY0Y50Y2A8M4A7E9NN8VE"]
-pub struct MapRespawnPoint(pub Vec3);
+pub struct MapSpawner(pub Entity);
```

Comment:
I think we need a way to make it clear that this will delete, and unhydrate the spawner whenever something with this component goes out of bounds.

Probably just re-naming it to `ReHydrateOutOfBounds` or something like that.